src/main/java/uk/gov/hmcts/darts/transcriptions/component/impl/TranscriberTranscriptsQueryImpl.java

```diff
@@ -12,11 +12,14 @@
 import java.util.List;
 
 import static uk.gov.hmcts.darts.common.enums.SecurityRoleEnum.TRANSCRIBER;
+import static uk.gov.hmcts.darts.transcriptions.enums.TranscriptionStatusEnum.WITH_TRANSCRIBER;
 
 @Component
 @RequiredArgsConstructor
 public class TranscriberTranscriptsQueryImpl implements TranscriberTranscriptsQuery {
 
+    public static final String USR_ID = "usr_id";
```

Comment:
Should these 2 constants be made private visibility?

Reply:
yes - I'm gonna blame this one on refactor -> extract constant :-)